cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -545,7 +559,7 @@ public boolean apply(Id.Program programId) {
     //Delete the schedules
     for (WorkflowSpecification workflowSpec : spec.getWorkflows().values()) {
       Id.Program workflowProgramId = Id.Program.from(appId, workflowSpec.getName());
-      scheduler.deleteSchedules(workflowProgramId, ProgramType.WORKFLOW);
+      scheduler.deleteSchedules(workflowProgramId, SchedulableProgramType.WORKFLOW);
```

Comment:
I guess I am confused because SchedulableProgramType has MapReduce and Spark, too.... but ok, that can come later.